src/Microsoft.AspNetCore.Http.Connections/WebSocketOptions.cs

```diff
@@ -10,6 +10,11 @@ public class WebSocketOptions
     {
         public TimeSpan CloseTimeout { get; set; } = TimeSpan.FromSeconds(5);
 
+        /// <summary>
+        /// Gets or sets a value that determines if websocket frames should be be preserved.
```

Comment:
If this is set in `MapHub`, you're going to have a very bad time. We should a) document that it can't be set when using Hubs and b) have SignalR throw if the feature is present and indicates that raw frames are being used.

Reply:
Yep. Will do.

Reply:
So this throws a wrench into things a bit. I think it makes sense to see if it's possible to wire this up some other way without it being an option on WebSockets.

Reply:
That was kinda my initial thought, and it was the SignalR problem that really clarified it for me.